core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -161,7 +162,7 @@ class UserService(protected val userInfo: UserInfo,
     }
   }
 
-  private def makeBillingProjectResponse(projectRoles: Set[ProjectRole], billingProject: RawlsBillingProject) =
+  def makeBillingProjectResponse(projectRoles: Set[ProjectRole], billingProject: RawlsBillingProject) =
```

Comment:
I wanted to be able to use this for tests (note though that I haven't attempted to get tests passing yet).

Reply:
I think most people are too quick to declare class members private, for exactly this reason.

Reply:
I ended up moving it to the case class so you can call it statically.